lang/en/auth.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
```

Comment:
Do we want to add these files to the phpcs path too?

Reply:
Done